code/modules/sound/sound_emitter.dm

```diff
@@ -14,6 +14,10 @@
 /atom
 	var/datum/sound_emitter/sound_emitter
 
+/atom/Destroy()
```

Comment:
Make this `/atom/movable` - non movable atoms such as turfs or walls cannot (and shouldn't) be qdel'd.

Reply:
On a more general level, probably (only a suggestion) worth using `return ..()` and, if you want, move it the first definition of `/atom/movable/Destroy()`, although I understand the stylistic choice here